### PR TITLE
Orgs import

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Adds an importer and fields for org data currently stored in map.ucf.edu. The org data is imported during the already built `wp locations import` command.

In addition, an admin js file has been added to accomplish the simple task of auto collapsing the organization rows within the admin screen by default. If you look at the Millican Hall post in dev, you can see it can get pretty overwhelming with all of the rows uncollapsed to begin with.

**Motivation and Context**
This will allow us to display the org data that is already available in map.ucf.edu currently.

**How Has This Been Tested?**
Changes are available in dev for review.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: importers/class-ucf-location-importer.php [hunk 1]
```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
...
+
+		/**
+		 * Capitolizes a string with respect for
+		 * stop words, like for, and and of.
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 * @param string $words The phrase to be capitolized
+		 * @return string
+		 */
+		private function smart_capitolize( $words ) {
```
Comment (cadie) [inline importers/class-ucf-location-importer.php]: `capitolize` is misspelled, it should be an 'a' instead of 'o'

File: importers/class-ucf-location-importer.php [hunk 2]
```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
...
+			// Start fresh with every import
+			$data = array();
+
+			foreach( $orgs as $i => $org ) {
+				$org_name = $org->name;
+				$org_phone = isset( $org->phone ) ? $org->phone : null;
+				$org_room = isset( $org->room ) ? $org->room : null;
+
+				$org_data = array(
+					'org_name'        => $this->smart_capitolize( $org_name ),
```
Comment (cadie) [inline importers/class-ucf-location-importer.php]: update function name here too

File: importers/class-ucf-location-importer.php [hunk 3]
```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
...
+
+				if ( isset( $org->departments ) && count( $org->departments ) > 0 ) {
+					foreach( $org->departments as $i => $dept ) {
+						$dept_name  = $dept->name;
+						$dept_phone = isset( $dept->phone ) ? $dept->phone : null;
+						$dept_build = isset( $dept->building ) ? $dept->building : null;
+						$dept_room  = isset( $dept->room ) ? $dept->room : null;
+
+						$dept_data = array(
+							'dept_name'     => $this->smart_capitolize( $dept_name ),
```
Comment (cadie) [inline importers/class-ucf-location-importer.php]: update function name here too

File: importers/class-ucf-location-importer.php [hunk 4]
```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
...
+				}
+
+				$data[] = $org_data;
+			}
+
+			update_field( 'ucf_location_orgs', $data, $post_id );
+		}
+
+		/**
+		 * Capitolizes a string with respect for
```
Comment (cadie) [inline importers/class-ucf-location-importer.php]:
```suggestion
		 * Capitalizes a string with respect for
```

File: importers/class-ucf-location-importer.php [hunk 5]
```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
...
+		 * stop words, like for, and and of.
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 * @param string $words The phrase to be capitolized
```
Comment (cadie) [inline importers/class-ucf-location-importer.php]:
```suggestion
		 * @param string $words The phrase to be capitalized
```

File: importers/class-ucf-location-importer.php [hunk 6]
```diff
@@ -407,6 +413,85 @@ private function update_terms( $post_id, $data ) {
...
+
+				$org_data = array(
+					'org_name'        => $this->smart_capitalize( $org_name ),
+					'org_phone'       => $org_phone,
+					'org_room'        => $org_room,
+					'org_departments' => array()
+				);
+
+				if ( isset( $org->departments ) && count( $org->departments ) > 0 ) {
+					foreach( $org->departments as $i => $dept ) {
```
Comment (cjg89) [inline importers/class-ucf-location-importer.php]: Probably doesn't matter since `$i` isn't used in either foreach loop, but there may be weirdness with `$i` being overwritten in this loop (previously defined on line 428)?

Again, not really a big deal since it's not utilized anywhere, but may need to be revisited if this code is ever changed to use `$i`.

Comment (jmbarne3) [reply importers/class-ucf-location-importer.php]: The `$i` iterators were there because initially I was using the `update_sub_field` function with the array selector: `update_sub_field( array( 'ucf_location_orgs', $i, 'sub_field_name'), $value, $post_id );`. I was only doing that because the `update_field` method wasn't working, but it was only not working because I have forgotten to include the `$post_id` in it. Just never removed the iterators after I switched back the other method.

TL;DR: I just removed the iterators.
